packages/snap-preact-components/src/components/Atoms/Loading/LoadingBar.tsx

```diff
@@ -44,21 +44,25 @@ const CSS = {
 
 export const LoadingBar = observer((properties: LoadingBarProps): JSX.Element => {
 	const globalTheme: Theme = useTheme();
-	const theme = { ...globalTheme, ...properties.theme };
 	const defaultProps: Partial<LoadingBarProps> = {
 		height: '5px',
 	};
 
 	const props = mergeProps('loadingBar', globalTheme, defaultProps, properties);
 
-	const { active, color, backgroundColor, height, disableStyles, className, style } = props;
+	const { active, disableStyles, className, style, styleScript } = props;
 
 	const styling: { css?: StylingCSS } = {};
-	if (!disableStyles) {
-		styling.css = [CSS.loadingBar({ height, color, backgroundColor, theme, animation: CSS.animation }), style];
+	const stylingProps = { ...props };
```

Comment:
Should we maybe find all places where we are doing this and just set `stylingProps = props` instead of spreading into a new object? Not a huge difference but when you are doing this across hundreds of components it could add up.   ... maybe...